packages/components/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
 	"extends": "../../tsconfig.base.json",
 	"compilerOptions": {
+		"importsNotUsedAsValues": "error",
```

Comment:
Let's move this to the base config at `tsconfig.base.json`. We want all packages to use it.